neurobooth_os/iout/iphone.py

```diff
@@ -703,6 +721,20 @@ def dump(self, filename: str, timeout_sec=None) -> (bool, ByteString):
                     self._raise_timeout("@DUMP")
             except IPhonePanic as e:
                 self.panic(e)
+
+            if file_hash is None:
+                self.logger.warning(f'No hash check performed for {filename}')
```

Comment:
I had originally implemented things such that the hash checks would be optional (depending on the calling script). This was partly because I wanted to code what I could before everything was in place. I'll just go ahead and make the hash checks manditory.